go.mod

```diff
@@ -3,9 +3,9 @@ module github.com/gnolang/gno
 go 1.20
 
 require (
-	github.com/btcsuite/btcd v0.22.0-beta.0.20220111032746-97732e52810c
+	github.com/btcsuite/btcd v0.22.3
 	github.com/btcsuite/btcd/btcutil v1.0.0
-	github.com/btcsuite/btcutil v1.0.2
+	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
```

Comment:
Ehh, I'd say upgrading btcd/btcutil is something that should be done in another PR, especially considering https://github.com/gnolang/gno/pull/1064

I'd suggest reverting changes here + go.sum

Reply:
It was accidental, sorry. Will revert them :)

Reply:
Oops, my mistake. I attempted to fix the previous commit but didn't thoroughly check it.